<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMenu</class>
 <widget class="QMainWindow" name="StartMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>331</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>331</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>422</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="gameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>게임</string>
    </property>
   </widget>
   <widget class="QPushButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>정보</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creditButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>CREDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ladderButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255,255,255);
front : 10pt &quot;MS Shell Dig 2&quot;;</string>
    </property>
    <property name="text">
     <string>랭킹</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
